tests/src/test/javascript/msltests.html

```diff
@@ -70,6 +70,7 @@
 <script type="text/javascript" src="../../../../core/src/main/javascript/crypto/WebCryptoUsage.js"></script>
 <script type="text/javascript" src="../../../../core/src/main/javascript/crypto/CipherKey.js"></script>
 <script type="text/javascript" src="../../../../core/src/main/javascript/crypto/PublicKey.js"></script>
+<script type="text/javascript" src="../../../../core/src/main/javascript/crypto/KeyFormat.js"></script>
```

Comment:
This file should be before PublicKey.js. Or even higher because asnjwk.concat.js references it. Or asnjwk.concat.js should be moved lower, since KeyFormat.js references MslCryptoException. Hm.